astro/src/content/docs/operate/deploy/terraform.mdx

```diff
@@ -0,0 +1,432 @@
+* The `fusionauth_default_tenant_id` variable can be omitted for now, but you'll update it later.
+
+<RemoteCode lang="hcl" title="Example Variable Definitions" url="https://raw.githubusercontent.com/fusionauth/fusionauth-example-terraform/main/examples/import/terraform.tfvars" />
+
+The variable definition with `terraform.tfvars` is only one option for setting these variables. Review the [Assigning Values to Root Module Variables](https://developer.hashicorp.com/terraform/language/values/variables#assigning-values-to-root-module-variables) documentation for more options.
+
+Next, add the [FusionAuth Terraform Provider](https://registry.terraform.io/providers/fusionauth/fusionauth/latest) configuration in the directory by:
+
+* Opening the [FusionAuth Terraform Provider Documentation](https://registry.terraform.io/providers/fusionauth/fusionauth/latest/docs).
+* Clicking the <strong>USE PROVIDER</strong> button.
+* Copying the Hashicorp Configuration Language (HCL) provided.
+* Creating a Terraform configuration file named `main.tf`.
+* Pasting the copied HCL into that file.
+* Adding the previously created variable as `var.fusionauth_api_key` to the `provider` section
+* Adding the FusionAuth instance you plan to manage with the `var.fusionauth_host` variable in the `provider` section
+
+Here's a screenshot of the page.
+
+![Downloading the HCL for the FusionAuth provider.](/img/docs/operate/deploy/terraform-provider.png)
+
+Here's what your `main.tf` file should look like at this point.
+
+<RemoteCode lang="hcl" title="Example main.tf File" url="https://raw.githubusercontent.com/fusionauth/fusionauth-example-terraform/main/examples/import/main.tf" tags="terraformProvider" />
+
+Finally, the following command prepares the current working directory for use with Terraform.
+
+```shell
+terraform init
+```
+
+It is always safe to run [`terraform init`](https://developer.hashicorp.com/terraform/cli/commands/init) multiple times, to bring the working directory up to date with changes in the configuration. Though subsequent runs may give errors, this command will never delete your existing configuration or state.
+
+## Choosing A Configuration Strategy
+
+Once you initialize the Terraform working directory, review different initial strategies to handle resource creation.
+
+| Strategy | Advantages | Disadvantages |
+| ---- | ---- | ---- |
+| import | manage existing resources | some resources don't fully support the Terraform lifecycle |
+| data source  | reference existing resources without having to manage them | requires a external process to manage these resources |
+| create | always create and be able to manage all resources through Terraform | bigger initial effort and default resources can't be managed |
```

Comment:
It may not have anything to do with this PR, but this table didn't render correctly for me. The column headers were not aligned above the row elements.

Reply:
This is due to https://github.com/FusionAuth/fusionauth-issues/issues/2556